test/tzdata/download.jl

```diff
@@ -1,23 +1,43 @@
-using TimeZones.TZData: tzdata_url, tzdata_download, isarchive, LATEST_FILE, read_latest
+using TimeZones.TZData: tzdata_url, tzdata_download, isarchive, LATEST_FILE, read_latest, REGIONS, latest_version,
+    tzdata_version
+using Dates
+if VERSION >= v"1.4"
+    using Pkg.Artifacts
+end
+
 
 @test tzdata_url("2016j") == "https://data.iana.org/time-zones/releases/tzdata2016j.tar.gz"
 @test tzdata_url("latest") == "https://data.iana.org/time-zones/tzdata-latest.tar.gz"
 
 # Note: Try to keep the number of `tzdata_download` calls low to avoid unnecessary network traffic
 mktempdir() do temp_dir
```

Comment:
I think these tests should just be for the non-artifact version

Reply:
Makes sense.